docs/user-guide/toc.md

```diff
@@ -120,26 +143,17 @@ Through this toolbar it is possible to:
 
 * [Create Widgets](widgets.md) for the selected layer <img src="../img/button/widgets.jpg" class="ms-docbutton"/>
 
-Selecting a group the toolbar is the following:
+* Access the **Compare tool** where it is possible *Swipe* or *Spy* the selected layer <img src="../img/button/compare_tool_button.jpg" class="ms-docbutton"/>. 
 
-<img src="../img/toc/group-toolbar.jpg" class="ms-docimage"/>
+From the dropdown menu <img src="../img/button/dropdown_menu_button.jpg" class="ms-docbutton" style="max-height:25px;"/> by clicking on <img src="../img/button/swipe_button.jpg" class="ms-docbutton" style="max-height:25px;"/> button a draggable handle appears on the map to swipe the selected layer. 
 
-In this case the user is allowed to:
+<img src="../img/toc/swipe_on_map.jpg" class="ms-docimage" style="max-width:600px;"/>
 
-* **Add layer to selected group** <img src="../img/button/add_layer_button.jpg" class="ms-docbutton"/>
+Once the Swipe is active, also on the dropdown menu <img src="../img/button/dropdown_menu_button.jpg" class="ms-docbutton" style="max-height:25px;"/>, by clicking on <img src="../img/button/configure_button.jpg" class="ms-docbutton" style="max-height:25px;"/> button the editor can change the direction of the swipe from *Vertical* to *Horizontal*.
 
-* **Add sub group to the selected group** <img src="../img/button/add_group_button.jpg" class="ms-docbutton"/>
+The user can also activated the <img src="../img/button/spy_button.jpg" class="ms-docbutton" style="max-height:25px;"/> from the same dropdown menu <img src="../img/button/dropdown_menu_button.jpg" class="ms-docbutton" style="max-height:25px;"/> and spy on the layer with the spy glass.
```

Comment:
The user can also activated the <img src="../img/button/spy_button.jpg" class="ms-docbutton" style="max-height:25px;"/> from the same dropdown menu <img src="../img/button/dropdown_menu_button.jpg" class="ms-docbutton" style="max-height:25px;"/> in order to switch the Compare tool in **Spy glass** mode. If the *Spy glass* is active, clicking on the <img src="../img/button/configure_button.jpg" class="ms-docbutton" style="max-height:25px;"/> option, the configuration modal opens so that it is possible to change the size of the spy glass (the `radius`).